src/internal/utils/auth.ts

```diff
@@ -0,0 +1,73 @@
+import {InvalidArgumentError} from '../../errors/errors';
+import jwtDecode from 'jwt-decode';
+import {isBase64} from './validators';
+import {decodeFromBase64} from './string';
+
+export interface LegacyClaims {
+  /**
+   * control plane endpoint
+   */
+  cp: string;
+  /**
+   * cache endpoint
+   */
+  c: string;
+}
+
+interface V1ApiTokenClaims {
+  sub: string;
+  ver: number;
+  exp: number;
+}
+
+export interface V1ApiTokenClaimsWithEndpoint extends V1ApiTokenClaims {
```

Comment:
Is this used anywhere?

Reply:
removed in https://github.com/momentohq/client-sdk-nodejs/pull/336/commits/eeebeddaade055970291a058ee31981e65436625